katello/katello-remove

```diff
@@ -129,13 +128,19 @@ CONTENT=[
   '/usr/share/katello-installer-base',
   '/usr/share/qpid',
   '/usr/share/qpid-tools',
-  '/var/cache/candlepin',
-  '/var/cache/foreman-proxy',
```

Comment:
`/var/cache/` is listed below so they were completely pointless

https://github.com/Katello/katello-packaging/blob/master/katello/katello-remove#L134